### DISCUSSION TITLE
Model for Unsupported Species

### DISCUSSION THREAD

**Discussion by katee9** [Open-Systems-Pharmacology/Forum#1552] [Questions & Problems]
Hello! Currently we are working on a PBPK model for our project, and our target species are cetaceans, which, based on my current knowledge, are not currently among the supported species.

Because of this, I would like to ask a few questions regarding modeling for currently unsupported species:

1. When selecting species under the individual menu, is it ok to just select any species since the body parameters will be changed later on?
2. For the body parameters such as compartment volumes and masses, cardiac output, etc., is there any way to input custom allometric equations for the relationships between these parameters to the body mass?
3. For the administration protocol, we are assuming that the cetacean individual will be exposed to the compound through its diet, and the amount of compound changes over time due to increase in dietary consumption (due to age). How can this be reflected under the administration protocol?

Any advice on the following would be appreciated.

Thank you!

**Comment by StephanSchaller** [comment]
Hi @katee9,

Yes, you can first create the model with any species and later substitute physiology by parameterization. WHile there is no in-built feature to do this via allometric scaling (at least I am not directly aware of @Yuri05 ?) you can populate the model through R and an excel sheet with the physiology information gathered (e.g. from literature) as was done e.g. for birds just recently: 
https://www.sciencedirect.com/science/article/pii/S0160412022004743
